web/src/Stylesheets.elm

```diff
@@ -12,7 +25,13 @@ port files : CssFileStructure -> Cmd msg
 fileStructure : CssFileStructure
 fileStructure =
     Css.File.toFileStructure
-        [ ( "../../stylesheets/main.css", Css.File.compile [ css ] ) ]
+        [ ( "../../stylesheets/main.css"
+          , Css.File.compile
+                [ generalCss
```

Comment:
it does look like there's one or two extra spaces in there, but that's how elm-format saves it so you'll have to find the bug in elm-format if you want to change it 😏